package.json

```diff
@@ -95,6 +95,7 @@
         "pg": "^8.11.3",
         "pino": "^8.15.1",
         "pino-pretty": "^10.0.0",
+        "posthog-node": "^4.1.0",
```

Comment:
there was a guard for null values added in 4.1.1.  also V4.2.0 is released.

We should probably switch to 4.1.1 at min.